plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryCaseInsensitiveMapping.java

```diff
@@ -35,15 +39,27 @@
 @Test(singleThreaded = true)
 public class TestBigQueryCaseInsensitiveMapping
         // TODO extends BaseCaseInsensitiveMappingTest - https://github.com/trinodb/trino/issues/7864
-        extends AbstractTestQueryFramework
+        extends BaseCaseInsensitiveMappingTest
 {
     private BigQuerySqlExecutor bigQuerySqlExecutor;
+    private Path Mappingfile;
+
+    @Override
+    protected SqlExecutor onRemoteDatabase() {
+        return requireNonNull(bigQuerySqlExecutor,"bigQuerySqlExecutor is null");
+    }
+
+    @Override
+    protected Path getMappingFile() {
+        return requireNonNull(Mappingfile,"Mappingfile is null");
```

Comment:
Remove the `requireNonNull` because `createRuleBasedIdentifierMappingFile` doesn't return nulls. It throws on failure.

Same in other files.